.changeset/silent-otters-relax.md

```diff
@@ -0,0 +1,10 @@
+---
+"@biomejs/biome": minor
+---
+
+Enable `.editorconfig` by default, default value of [`formatter.useEditorconfig`](https://biomejs.dev/reference/configuration/#formatteruseeditorconfig) set to `true`.
+
+It will follow the following rules:
+
+- When `biome.json` and `.editorconfig` in the same directory, formatter settings from `biome.json` take precedence.
+- If `biome.json` is not present or it exists without formatter settings, and an `.editorconfig` file exists in a directory higher up in the hierarchy, then default formatter settings should be used and the `.editorconfig` file should be ignored.
```

Comment:
These are correct, but maybe I'd rephrase it a little bit for clarify to our users:

```suggestion
- Formatting settings in `biome.json` always take precedence over `.editorignore` files.
- `.editorignore` files that exist higher up in the hierarchy than a `biome.json` file are always ignored. This is to avoid loading formatting settings from someone's home directory into a project with a `biome.json` file.
- We currently don't support nested `.editorignore` files.
```

Reply:
done